webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -14,19 +16,15 @@ class IntlLoader {
 
   async init() {
     await this.fetchIntl();
-    const localeData = await import(
```

Comment:
Since I have almost 0 knowledge about JS world, I searched "difference between import and require in JS" and looking at the answers it seems that this file shouldn't be working at first place?..

I mean, they say `import` can't be used conditionally and anywhere expect the beginning of a file. Also, they say that `require` is synchronous, so we don't need `async/await` here?..

Reply:
You're right, removed the await